dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -340,6 +338,8 @@ private void addTrackedEntityFromItem(
       TrackedEntityQueryParams params,
       PageParams pageParams,
       boolean isCountQuery) {
+    SqlHelper sqlHelper = new SqlHelper(true);
```

Comment:
This variable definition could remain closer to its first use to help readability.